cmd/cli/operations/update_job.go

```diff
@@ -24,7 +24,7 @@ type UpdateJob struct {
 	AllowNonRestoredState bool
 }
 
-func (o RealOperator) filterRunningJobs(jobs []flink.Job) (ret []flink.Job) {
+func (o RealOperator) filterRunningJobsByName(jobs []flink.Job, jobNameBase string) (ret []flink.Job) {
```

Comment:
You added the `jobNameBase` to the function parameters but you're not actually performing any filtering on it. The function still does exactly the same as before

Reply:
OMG, whole bunch of UT changes were missing. I guess my merge went way wrong from my local to my fork. Please see my latest commit.